Integrations/build.gradle

```diff
@@ -61,6 +61,9 @@ def runInDocker = { String name, String sourcePath, List<String> command, Closur
             from(test.runtimeClasspath) {
                 into 'classpath'
             }
+            from("${sourcePath}/../client") {
+                into 'python/client'
+            }
```

Comment:
This seems really fragile, and likely to cause cache invalidation due to changes in the `py/client/build` dir. At a minimum, we should probably only `include` the files necessary?

Reply:
Alternatively, sould we depend on the wheel output instead? You should be able to do something like:

```gradle
configurations {
    pythonWheel
}

dependencies {
    pythonWheel project(':py-client')
}
...
    copyIn {
        from(configurations.pythonWheel) {
            into 'py-client-wheel'
        }
    }
```

see py/client-ticking for example.